### DISCUSSION TITLE
Screen Time

### DISCUSSION THREAD

**Discussion by Louns-stack** [elrumo/macOS_Big_Sur_icons_replacements#1091] [Help with the website]
Hello, I already have 2 questions,
I'm sorry if you've already answered, but I've searched for a long but found nothing.
Actually, I just don't know if it is normal that my screen-time became fully used after using those icons. (CF screenshot number 1) 
Secondly, is there any way to upload the original icons ? 
<img width="224" alt="Capture d’écran 2021-02-14 à 17 59 19" src="https://user-images.githubusercontent.com/79054567/107883291-5e66cf80-6eee-11eb-894f-1ff13836e73f.png">
Thanks a lot !

**Comment by artexhibit** [comment]
Have the same issue. Each time after the web site visit. Need to reboot Mac to fix that behaviour.

**Reply by Louns-stack** [reply]
Oh wel, thanks, I hope it will be fixed soon, and do you know the answer ton my second question all chance ? 😰

**Reply by artexhibit** [reply]
> Oh wel, thanks, I hope it will be fixed soon, and do you know the answer ton my second question all chance ? 😰

Yeap, me too. Can you specify what do you mean about original icons upload?

**Reply by Louns-stack** [reply]
Such as restoring the icons that you had before changing it

**Reply by artexhibit** [reply]
> Such as restoring the icons that you had before changing it 
> 
> 
Hmm, when you replace icon, the original one is replaced by a new one. As I can see the only way to restore it either reinstall app or pre-save original icon. 

You can do it easily: right-click the app from within the Finder, select Show Package Contents, then contents -> Resource. 

There you can find the original icns file and copy it.

**Reply by Louns-stack** [reply]
Thanks a lot for you're advices ! Such an helpful guy you are ! 😇

**Reply by elrumo** [reply]
An even easier way to restore the icon is to select the app on finder, "Get info", click on the icon and press the backspace key

**Comment by elrumo** [comment]
To your first question, what browser are you using? I couldn't reproduce the error, does that happen instantly after you visit the website?

**Reply by Louns-stack** [reply]
First of all, thanks for you're first tip, it works perfectly !! And actually on safari, I'm just doing the test

**Reply by artexhibit** [reply]
> To your first question, what browser are you using? I couldn't reproduce the error, does that happen instantly after you visit the website?

From Safari for me. And yes, this behavior starts right after I downloaded some icons for example and leave the site
